discovery/seata-discovery-raft/src/main/java/io/seata/discovery/registry/raft/RaftRegistryServiceImpl.java

```diff
@@ -315,14 +352,28 @@ private static void acquireClusterMetaDataByClusterName(String clusterName) {
 
     private static void acquireClusterMetaData(String clusterName, String group) {
         String tcAddress = queryHttpAddress(clusterName, group);
+        Map<String, String> header = new HashMap<>();
+        String token = getToken(tcAddress);
+        if (!Objects.isNull(token)) {
```

Comment:
放到初始化的时候拿token，而不是在请求的时候拿token，token拿不到直接抛异常阻止应用启动
Put in the initialization time to get the token, not in the request time to get the token, the token can not get directly throw an exception to prevent the application startup

Reply:
fixed